### PR TITLE
Chapter 3: Peano Axioms

### PR DESCRIPTION
**Pull request by wei2912**
I made a few changes to the text talking about Peano Axioms. If I have time, I might add the section on addition and multiplication.

### CODE REVIEW

File: book/chapters/3-special-sets.ltx [hunk 1]
```diff
@@ -118,13 +118,14 @@ $\{0, 1, 2, 3, 4, 5, 6, \ldots\}$.
...
   
-\item[Axiom 9] Let's say $K$ is a set of numbers (a bunch of numbers). If we
-  know that
+\item[Axiom 9] Let's say $K$ is a set of numbers (a bunch of numbers). If $K$ is a set such that:
   \begin{enumerate}
-  \item if $0$ is in $K$, and
-  \item if some number $x$ is in $K$, then $\suc(x)$ is in $K$,
-  \item then $K$ contains every single natural number.
+  \item $0$ is in $K$, and
+  \item for every natural number x, if x is in K, then suc(x) is in K,
```
Comment (pharpend) [inline book/chapters/3-special-sets.ltx:124]: Can you add math formatting for this line? Then I'll merge.

**Comment by pharpend** [comment]
Er, no, I don't like these changes. All you've done is make the language more formal, and you screwed up some of the formatting. Rejecting.

**Comment by ghost** [comment]
@pharpend can you give a more specific feedback so @wei2912 can possibly amend the merge?

**Comment by wei2912** [comment]
On second thought, this is perfectly fine. My linguistic skills are horrible at night.

**Pull request closed without merging**